### PR TITLE
What: Slack governance document

### PR DESCRIPTION
**Pull request by TheFoxAtWork**
Why:

* Provide governance on posting to the slack channels
* provide identifiers for finding slack channels

This change addresses the need by:

* providing a slack.md in the root of the repo
* linking the slack.md to the README under the communications subheading

### CODE REVIEW

**Comment by caniszczyk** [comment]
Also recommend linking to https://github.com/cncf/foundation/blob/master/slack-guidelines.md

File: slack.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# SIG-Security channels housekeeping
+
+## Identifying and creating channels
+Just for approved projects, sec-assessment-xxxx exception SIG-Security channels
+are identified with the “sig-security-” prefix.  Except during conferences, the
+CNCF permits slack members to create channels; however, security-related
```
Comment (lumjjb) [inline slack.md:6]:
```suggestion
CNCF permits slack members to create channels; however, sig-security-related
```

File: slack.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+channels should only be created by chairs or tech leads, and are typically
+prefixed by sig-security- following hyphenation of the topic/subject.  This
```
Comment (lumjjb) [inline slack.md:8]:
```suggestion
prefixed by "sig-security-" following hyphenation of the topic/subject.  This
```

File: slack.md [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+Members of SIG-Security channels are expected to abide by the [code of conduct](https://github.com/cncf/sig-security/blob/master/CODE-OF-CONDUCT.md).
+
+## Posting oputside content
+
+The SIG-Security channels are mechanisms for cloud native security discussions.
+It is expected that outside, non-sig created content will be posted; however,
+these should include topics of relevance and interest to the cloud native
+community space, rather than marketing or promotion of a vendor-specific
+product..
```
Comment (lumjjb) [inline slack.md:26]:
```suggestion
product.
```

File: slack.md [hunk 4]
```diff
@@ -0,0 +1,30 @@
+# SIG-Security channels housekeeping
+
+## Identifying and creating channels
+Just for approved projects, sec-assessment-xxxx exception SIG-Security channels
```
Comment (lumjjb) [inline slack.md:4]:
```suggestion
Just for approved projects, "sec-assessment-xxxx" exception SIG-Security channels
```

**Review by lumjjb — APPROVED** [review]
Couple nits, if not LGTM

File: slack.md [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+
+It is requested that channels include a header for what it is used for.
+
+Additional information may be found in the [CNCF slack guidelines](https://github.com/cncf/foundation/blob/master/slack-guidelines.md).
+
+## Code of conduct
+
+Members of SIG-Security channels are expected to abide by the [code of conduct](https://github.com/cncf/sig-security/blob/master/CODE-OF-CONDUCT.md).
+
+## Posting oputside content
```
Comment (deedubs) [inline slack.md:20]:
```suggestion
## Posting outside content
```

**Comment by TheFoxAtWork** [comment]
done

**Pull request merged**